styles/books/neuroscience/book.scss

```diff
@@ -124,13 +124,30 @@ $ChapterIntroType: side;
   ),
 ));
 
+// EOB
+@include add_settings((
+  AppendixGlossary: (
+    _selectors: (".appendix > .column-container"),
+    'H3AppendixGlossaryTitle:::color': (_ref: "colorMap:::neuroscienceHeaderColor"),
+    'AppendixGlossaryContainer:::column-count': 2,
+  ),
+));
+
 // Book specific styles
 @include use('BookRoot', "common:::BookRoot");
 @include use('LearningObjectivesNumbered', 'carnival:::LearningObjectivesNumberedShape');
 
 // TOC
 @include use('TOC', 'carnival:::NoUnitToc');
 
+// EOB
+@include add_settings((
+  AppendixGlossary: (
+    _selectors: (".appendix > .column-container"),
+    'H3AppendixGlossaryTitle:::color': (_ref: "colorMap:::neuroscienceHeaderColor"),
+    'AppendixGlossaryContainer:::column-count': 2,
+  ),
+));
```

Comment:
looks like AppendixGlossary `add_settings` is duplicated

Reply:
Ha, yeah Jen and I were both working on it i think there was some collision :)